frontend/app/forms/application-for-pardon/application-for-pardon.web.component.js

```diff
@@ -0,0 +1,202 @@
+          <TextInput
+            dataKey="person.socialSecurity"
+            label={__("social security number")}
+          />
+          <TextInput
+            dataKey="person.driversLicenseNumber"
+            label={__("drivers license #")}
+          />
+          <TextInput
+            dataKey="person.driversLicenseState"
+            label={__("drivers license state")}
+          />
+          <TextInput
+            dataKey="person.driversLicenseState"
+            label={__("drivers license state")}
+          />
+          <TextInput
+            dataKey="person.offenderNumber"
+            label={__("offender #")} // Offender #
+          />
+          <TextInput
+            dataKey="person.fbiNumber"
+            label={__("fbi #")} // FBI #
+          />
+          <TextInput dataKey="person.homePhone" label={__("home phone")} />
+          <TextInput dataKey="person.dayPhone" label={__("day phone")} />
+          <TextInput dataKey="person.email" label={__("email address")} />
+        </Section>
+
+        <Section name="2. Pardon Request and Consideration">
+          <Radio
+            dataKey="person.previouslyApplied"
+            label={__("previously applied")}
+            options={previouslyAppliedOptions}
+          />
+
+          <p className="web-form-input">
+            If yes, please provide the date of the application, the outcome of
+            the application, list any conviction(s) pardoned, and attach a copy
+            of the application and the Pardon Certificate.
+          </p>
+          {data.person.previouslyApplied && (
+            <TextArea
+              dataKey="person.explaination" // needs more specific name
+              label={__("previously please explain")}
+            />
+          )}
+        </Section>
+        <Section name="3. Family History">
+          <p className="web-form-input">
+            If married, please provide the following information for your spouse
+          </p>
+          <TextInput
+            dataKey="spouse.firstName"
+            label={__("first name spouse")}
+          />
+          <TextInput
+            dataKey="spouse.middleName"
+            label={__("middle name spouse")}
+          />
+          <TextInput dataKey="spouse.lastName" label={__("last name spouse")} />
+          <TextInput dataKey="spouse.addressStreet" label={__("street")} />
+          <TextInput dataKey="spouse.addressCity" label={__("city")} />
+          <TextInput dataKey="spouse.addressState" label={__("state")} />
+          <TextInput dataKey="spouse.addressZip" label={__("zip")} />
+          <TextInput dataKey="spouse.homePhone" label={__("home phone")} />
+          <TextInput dataKey="spouse.dayPhone" label={__("day phone")} />
+          <TextInput dataKey="spouse.email" label={__("email address")} />
+          // children of person
+        </Section>
+        <Section name="4. Employment History">
```

Comment:
See related https://github.com/JustUtahCoders/utahexpungements.org/blob/master/frontend/app/forms/motion-to-waive-fees/motion-to-waive-fees.web.component.js#L229 when you start working on this. There might be some dataKeys that can be reused there.